frontend/src/components/sightingCard/sightingCard.tsx

```diff
@@ -43,15 +37,14 @@ export const SightingCard: React.FunctionComponent<SightingCardProps> = ({
 
   let speciesSection = <></>;
   if (Array.isArray(species)) {
-    console.log(species.map);
     speciesSection = (
       <>
         {species.map((s) => {
           return (
-            <>
-              <p>Species Name: {s.name}</p>
-              <p>Species Description: {s.description}</p>
-            </>
+            <ul key={s.id}>
+              <li>Species Name: {s.name}</li>
+              <li>Species Description: {s.description}</li>
+            </ul>
```

Comment:
Now you're returning a bunch of `<ul>`s one after the other! The idea is that each species is in a list, so the result should be a list, with each element being an individual species, like
```
<ul>
  <li>
    <p>Species name</p>
    <p>Description</p>
  </li>
  <li>
    <p>Species name</p>
    <p>Description</p>
  </li>
  ...
</ul>
```
where you currently have
```
<ul>
  <li>Species name</li>
  <li>Description</li>
</ul>
<ul>
  <li>Species name</li>
  <li>Description</li>
</ul>
...
```
which, I'm afraid, doesn't make sense semantically.